include/aikido/constraint/outcome/CollisionFreeOutcome.hpp

```diff
@@ -0,0 +1,27 @@
+#ifndef AIKIDO_CONSTRAINT_COLLISIONFREEOUTCOME_HPP_
+#define AIKIDO_CONSTRAINT_COLLISIONFREEOUTCOME_HPP_
+
+#include <sstream>
```

Comment:
Could you elaborate on this a bit? If <sstream> is not included, then it looks like the string building in toString won't work (since a string stream is used there, and then converted to a string object).

Reply:
`sstream` is not exposed in this header file but used in the source file. So I suggest we include `<string>` as `std::string` is explicitly exposed in this header and include `<sstream>` in the source file where it's actually used. 

The motivation behind this is that this way prevents unnecessarily including `<sstream>` by including `CollisionFreeOutcome.hpp`.